src/combineReducers.js

```diff
@@ -74,6 +74,56 @@ function assertReducerSanity(reducers) {
   })
 }
 
+function finalReducers(reducers) {
```

Comment:
Not saying we want to get this in yet, but in any case the minimal requirement would be that we use verbs for function names.